.pre-commit-config.yaml

```diff
@@ -1,63 +1,60 @@
 repos:
   # api
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.2.1
+    rev: v0.4.4
     hooks:
-      - id: ruff
-        name: api|ruff-lint
-        args:
-          - --config=api/pyproject.toml
-          - --fix
-          - --fixable=I
-        files: ^api
       - id: ruff-format
         name: api|ruff-format
-        args: [--config=api/pyproject.toml]
+        entry: ruff format --config=api/pyproject.toml
         files: ^api
+        types: [python]
+      - id: ruff
+        name: api|ruff-check
+        entry: ruff check --config=api/pyproject.toml --fix --fixable=I
+        files: ^api
+        types: [python]
 
   # deployment
-  - repo: local
+  - repo: https://github.com/antonbabenko/pre-commit-terraform
+    rev: v1.89.1
     hooks:
-      - id: terraform-fmt
+      - id: terraform_fmt
```

Comment:
?

Reply:
J'ai préféré standardiser le mode d'application des pre-commits:
- comme on est dans un multirepo, il semblait sain de faire en sorte que pre-commit gère ses dépendances indépendamment d'in fichier de requirements
- donc j'ai retiré le mode local et installé `pre-commit-terraform` et l'ID du hook à appeler est `terraform_fmt` .